styles/style.css

```diff
@@ -32,12 +61,222 @@ img {
   width: 100%;
 }
 
-/* Navigation Section */
+ul {
+  list-style: none;
+}
+
+a {
+  transition: all 0.35s ease;
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: none;
+}
+
+/*--------------------
+ 3. BUTTONS
+-----------------*/
+.btn {
+  cursor: pointer;
+  display: inline-block;
+  line-height: 2rem;
+  border-radius: 4rem;
+  border: 2px solid transparent;
+  transition: all 0.4s ease;
+  font-weight: 700;
+}
+
+.btn-link {
+  background-color: var(--clr-white);
+  color: var(--clr-black);
+  padding: 1rem 2.2rem;
+  font-size: 1.4rem;
+  position: relative;
+}
+
+.btn-link:hover {
+  background-color: var(--clr-bg-blue);
+  color: var(--clr-white);
+  border-color: var(--clr-white);
+}
+
+.btn-link:hover i {
+  color: var(--clr-white);
+}
+
+/*--------------------
+ 4. NAVIGATION CSS STYLE
+-----------------*/
+.nav {
+  background-color: var(--clr-bg-blue);
+  color: var(--clr-white);
+  min-height: 10vh;
+  border-bottom: 0.1rem solid var(--clr-white);
+  position: fixed;
+  width: 100%;
+  z-index: 5;
+}
+
+.container-nav {
+  padding: 2rem 0rem;
+  display: grid;
+  grid-template-columns: 1fr auto minmax(800px, 3fr) 1fr 1fr;
+}
+
+.brand {
+  grid-column: 2 / 3;
+}
+
+.brand .brand-img {
+  max-width: 15rem;
+  min-width: 10rem;
+}
+
+.menu-links {
+  grid-column: 3 / 5;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.menu-link {
+  color: var(--clr-white);
+  text-transform: capitalize;
+  padding: 0 2rem;
+  font-weight: 600;
+}
+
+.menu-links a i {
+  margin-right: 3px;
+  color: var(--clr-icons);
+}
+
+/* Do not display the toggler and label toggler */
+input[type="checkbox"] {
+  display: none;
+}
+.toggler-label {
+  display: none;
+}
+
+/* Hamburger Menu Media Query */
+@media screen and (max-width: 960px) {
+  /* Set the logo position */
+  .container-nav {
+    display: grid;
+    grid-template-columns:
+      minmax(1rem, 1fr)
+      repeat(5, 1fr)
+      minmax(1rem, 1fr);
+    gap: 0.5rem;
+  }
+  .brand {
+    grid-column: 2/4;
+  }
+
+  /* Checkbox for hamburger. Set to absolute, rather than display:none to allow tabbability  */
+  .toggler {
+    display: absolute;
+    top: -9999px !important;
```

Comment:
If you need to add `!important` high chance you are doing something wrong somewhere, fix that and take out the important flag

Reply:
Ok. Will fix that